ejb-remote/pom.xml

```diff
@@ -20,7 +20,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.jboss.quickstarts.eap</groupId>
-    <artifactId>jboss-ejb-remote-parent</artifactId>
```

Comment:
Is this change from jboss-ejb-remote-parent to jboss-ejb-remote really expected ?